views/js/controller/routes.js

```diff
@@ -20,6 +20,7 @@ define(function(){
             'actions' : {
                 'index' : 'controller/Delivery/index',
                 'monitoring' : 'controller/Delivery/monitoring',
+                'monitoringAll' : 'controller/Delivery/monitoring',
```

Comment:
I shall create a separate controller in another PR once this PR and https://github.com/oat-sa/extension-tao-proctoring/pull/15 are merged.